$OpenBSD: patch-configure,v 1.8 2018/01/22 06:28:00 bcallah Exp $

- Always link with -pthread for libffi (XXX not needed any more?)
- Link with libc++abi on arm (to access unwind symbols)

Index: configure
--- configure.orig
+++ configure
@@ -18522,6 +18522,15 @@ $as_echo "#define SYMBOL_UNDERSCORE 1" >>confdefs.h
 
 fi
 
+case "$target" in
+    arm-*-openbsd*)
+      LDFLAGS="-lpthread -lc++abi $LDFLAGS"
+    ;;
+    *-*-openbsd*)
+      LDFLAGS="-lpthread $LDFLAGS"
+    ;;
+esac
+
 FFI_EXEC_TRAMPOLINE_TABLE=0
 case "$target" in
      *arm*-apple-darwin*)
